src/Essentials/src/AppInfo/AppInfo.shared.cs

```diff
@@ -25,5 +25,7 @@ public static partial class AppInfo
 
 		/// <include file="../../docs/Microsoft.Maui.Essentials/AppInfo.xml" path="//Member[@MemberName='RequestedTheme']/Docs" />
 		public static AppTheme RequestedTheme => PlatformRequestedTheme();
+
+		public static bool IsPackagedApp => PlatformIsPackagedApp();
```

Comment:
This is a new API and we may want to change this a bit. Not sure if this is the right name, but basically represents a single package that is "installed" on the OS using the OS "store" or sideloading. This does not mean that using some single file, .msi installer magically makes it work. It means that the app has no package identity. Probably need a new name tbh.

Reply:
An example is that a macOS/Cocoa app might be packaged but a WPF app will never be. Unless the rules change if the app is published to the store as a Win32 app. Then it probably gets a package identifier.

Reply:
Packaged isn't a bad choice.   I also think something along the lines of what the "App Model" is.  Should this be more than a bool, and maybe an enum?

Reply:
ooh, sounds like a good idea there.

Reply:
Is there a third model? There doesn't _have_ to be one to use an enum, but is it possible to have a third? The 2 modes I can think of: "in a self contained package with an ID" and "just a folder on disk"

I tested the APIs with a WinForms app and the Windows Application Packaging Project, and that uses the same native APIs that throw when run as an exe and has an ID when using the packaging project.